src/features/unified-share-modal/SharedLinkPermissionMenu.scss

```diff
@@ -0,0 +1,10 @@
+.ums-share-permissions-menu {
+    .ums-share-permissions-menu-item {
+        display: flex;
+        align-items: center;
+
+        .ftux-editable-shared-link {
+            margin-left: 10px;
```

Comment:
Should this only be for the `ftux-editable-shared-link` or less specific for other ftux enhancements?

Reply:
I think this is only for the `ftux-editable-shared-link` use case - for other ftux enhancements there will probably be a specific selector to apply that styling.